### PR TITLE
bump version and add --version|-v argument to print current version

### PR DESCRIPTION
**Pull request by ElvinEfendi**
@Shopify/traffic

### CODE REVIEW

File: lib/doggy/cli.rb [hunk 1]
```diff
@@ -40,7 +47,7 @@ def push(*ids)
       CLI::Push.new.push_all(ids)
     end
 
-    desc "mute IDs", "Mutes given monitors indefinitely"
+    desc "mute IDs [--duration <duration>]", "Mutes given monitors indefinitely or for the specific duration"
```
Comment (ElvinEfendi) [inline lib/doggy/cli.rb]: This was just an extra improvement I decided to include in this PR.

File: lib/doggy/cli.rb [hunk 2]
```diff
@@ -6,6 +6,13 @@ module Doggy
   class CLI < Thor
     include Thor::Actions
 
+    map %w[--version -v] => :__print_version
```
Comment (klautcomputing) [inline lib/doggy/cli.rb]: nitpicking here but `%w` uses `( )`

File: lib/doggy/cli.rb [hunk 3]
```diff
@@ -6,6 +6,13 @@ module Doggy
...
+
+    desc '--version, -v', 'print the version'
```
Comment (klautcomputing) [inline lib/doggy/cli.rb]: nitpicking again: all other strings in this seem to use `"` instead of `'`

Comment (ElvinEfendi) [inline lib/doggy/cli.rb]: yeah legacy thing from the previous contributor, I'm moving them to default to `'` slowly.

**Pull request merged**